### ISSUE TITLE
Error while trying to use `previewFormat` with AOT

### ISSUE THREAD

**Issue by OwenKelvin** [FrancescoBorzi/ngx-duration-picker#323]
The package has successfully installed and is perfectly running but when i try to add `previewFormat`  in the options i get below error

> core.js:4197 ERROR Error: Angular JIT compilation failed: '@angular/compiler' not loaded!
> - JIT compilation is discouraged for production use-cases! Consider AOT mode instead.
>  - Did you bootstrap using '@angular/platform-browser-dynamic' or '@angular/platform-server'?
> - Alternatively provide the compiler with 'import "@angular/compiler";' before bootstrapping.
>    at getCompilerFacade (core.js:562)
    at Function.get (core.js:26205)
    at getComponentDef (core.js:1826)
    at extractDirectiveDef (core.js:1650)
    at Array.map (<anonymous>)
    at def.directiveDefs (core.js:1633)
    at createTView (core.js:7406)
    at getOrCreateTComponentView (core.js:7355)
    at createRootComponentView (core.js:13413)
    at ComponentFactory$1.create (core.js:22149)

My Code looks like below
```     
    <ngx-duration-picker
          formControlName="period"
          [options]="{ previewFormat: '{{ H }}', customOutputFormat: 'H', showMonths: false, showSeconds: false, showYears: false, showWeeks: false }"></ngx-duration-picker>
```

**Comment by FrancescoBorzi** [comment]
hi @OwenKelvin I've just released the new version `3.3.0` of this library and updated all dependencies (including Angular).

Please try again with the latest version, and let me know if the bug persists.

**Comment by FrancescoBorzi** [comment]
hi @OwenKelvin I've just released the new version `3.3.1` of this library and updated all dependencies (including Angular).

Please try again with the latest version, and let me know if the bug persists.
